### PR TITLE
added variable for alternative service http

### PR DESCRIPTION
**Pull request by Ayush-Thakur-geek**
For issue #7373

### CODE REVIEW

**Comment by kluever** [comment]
Fixed via https://github.com/google/guava/commit/503ba429f9fbd4d2ef9a4c0ea28ccf62bed36afc

File: guava/src/com/google/common/net/HttpHeaders.java [hunk 1]
```diff
@@ -37,6 +37,8 @@ private HttpHeaders() {}
 
   // HTTP Request and Response header fields
 
+    /** The HTTP {@code Accept} header field name. */
+  public static final String ALT_SVC =  "Alt-Svc";
```
Comment (AnnieFraz) [inline guava/src/com/google/common/net/HttpHeaders.java:41]: Are there any tests surrounding this?

Comment (Ayush-Thakur-geek) [reply guava/src/com/google/common/net/HttpHeaders.java:41]: no

**Comment by jbduncan** [comment]
Given this PR has been merged in already, I propose we close it for now.

**Pull request closed without merging**